tests/operator/operator.go

```diff
@@ -3353,3 +3345,10 @@ func createRunningVMIs(vmis []*v1.VirtualMachineInstance) []*v1.VirtualMachineIn
 
 	return newVMIs
 }
+
+func deleteAllVMIs(vmis []*v1.VirtualMachineInstance) {
```

Comment:
you dropped the `All` from the create function, consider dropping it here.

Reply:
Not sure about it. The rename wasn't because of the "All" part of the name, but because the previous name didn't describe what the function is doing.